src/SPC/builder/linux/LinuxBuilder.php

```diff
@@ -109,6 +109,18 @@ public function makeAutoconfArgs(string $name, array $libSpecs): string
      */
     public function buildPHP(int $build_target = BUILD_TARGET_NONE): void
     {
+        if ($build_target === BUILD_TARGET_EMBED &&
+            file_exists(BUILD_BIN_PATH . '/php-config') &&
+            file_exists(BUILD_BIN_PATH . '/phpize')
+        ) {
+            $embed_type = getenv('SPC_CMD_VAR_PHP_EMBED_TYPE') ?: 'static';
+            if ($embed_type === 'shared' && file_exists(BUILD_LIB_PATH . '/libphp.so')) {
+                return;
+            }
+            if (file_exists(BUILD_LIB_PATH . '/libphp.a')) {
+                return;
+            }
+        }
```

Comment:
Do we have a scenario where we need to skip the build?

Reply:
What do you think? Skip php build when doing `bin/spc build "" --build-shared="..."` or never skip build?

Reply:
The ideal build should be a build lock file. The parts of the same dependency tree nodes are automatically skipped, and rebuilt if there are changes.

But we currently never ignore building PHP and extensions, only ignore dependent libraries, which helps to quickly test different extension combinations and save time.

Reply:
So I think we should not ignore all the build processes related to PHP and extensions including libphp.so.